examples/siri_exm_SX/exx_situationExchange_Pt.xml

```diff
@@ -24,12 +27,14 @@
  <siri:QualityIndex>certain</siri:QualityIndex>
  <siri:ValidityPeriod>
   <siri:StartTime>2001-12-17T09:30:47.0Z</siri:StartTime>
+  <siri:EndTime>2001-12-17T10:30:47.0Z</siri:EndTime>
  </siri:ValidityPeriod>
- <siri:MiscellaneousReason>bombExplosion</siri:MiscellaneousReason>
+ <siri:TpegReason>bombExplosion</siri:TpegReason>
  <siri:Severity>severe</siri:Severity>
+ <siri:Priority>3</siri:Priority>
  <siri:Audience>public</siri:Audience>
  <siri:ReportType>point</siri:ReportType>
- <siri:Summary overridden="true">Bomb at Barchester station</siri:Summary>
+ <siri:Summary overridden="true" xml:lang="EN">Bomb at Barchester station</siri:Summary>
  <siri:Description overridden="true" xml:lang="EN">Building evacuated. AVoid station untile furtehr notice</siri:Description>
```

Comment:
Please remove the "e" from "untile" and replace "furtehr" by "further".
And "AVoid" should probably be "Avoid".

Reply:
Thanks, resolved this!